README.md

```diff
@@ -1 +1,13 @@
-# vagrant-is
\ No newline at end of file
+# WSO2 Identity Server Vagrant Resources
+
+This repository contains the following resources:
+
+- Vagrant box generation
```

Comment:
can be better named as "Resources to generate Vagrant boxes for WSO2 Identity Server with Analytics"

Reply:
+1